### PR TITLE
Implement Done button, replace icons and do some refactor

### PR DESCRIPTION
**Pull request by tpai**
@kallepersson 
1. Use operator from #20 to implement Done button.
    screen shot 2018-10-09 at 5 36 48 pm

1. Replace icon with bigger one.
    before
    screen shot 2018-10-09 at 5 38 46 pm

    after
    screen shot 2018-10-09 at 5 39 18 pm

1. Fix toolbar background color, it should not change to gray when unfold the email.
    before
    
    after

### CODE REVIEW

File: in.js [hunk 1]
```diff
@@ -43,12 +66,24 @@ const reorderMenuItems = () => {
...
       observer.disconnect();
     }
   });
   observer.observe(document.body, {subtree:true, childList:true});
 };
 
+const activeMenuItem = (target, nodes) => {
+  nodes.map(node => node.firstChild.classList.remove('nZ'));
+  target.firstChild.classList.add('nZ');
+};
```
Comment (tpai) [inline in.js]: Once remove the id from element, the active style will not be added to it, so I create the active toggle function.

File: in.js [hunk 2]
```diff
@@ -3,34 +3,57 @@ _styleElement.innerText = _css;
...
+    ) {
       /* Gmail will execute its script to add element to the first child, so
        * add one placeholder for it and do the rest in the next child.
        */
       const placeholder = document.createElement('div');
       placeholder.classList.add('TK');
       placeholder.style.cssText = 'padding: 0; border: 0;';
 
+      // Assign link href which only show archived mail
+      done.querySelector('a').href = 'https://mail.google.com/mail/u/0/#search/-is%3Ainbox';
```
[the remark this answers is not in the document]
Comment (kallepersson) [reply in.js]: Great! Here I would only set href to `#search/-is%3Ainbox` as users might actually have multi-user gmail accounts eg `/u/1`

**Comment by kallepersson** [comment]
Thanks for this @tpai! Will review more later today

File: in.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (tpai) [reply in.js]: Exactly.

File: in.css [hunk 3]
```diff
@@ -39,10 +39,10 @@ body._in,
...
 /* Toolbar background */
 ._in header { background-color: #898984 !important; }
-._in[data-hash="#inbox"] header { background-color: #4285f4 !important; }
-._in[data-hash="#starred"] header { background-color: #FDB500 !important; }
-._in[data-hash="#snoozed"] header { background-color: #ef6c00 !important; }
-._in[data-hash="#all"] header { background-color: #0f9d58 !important; }
+._in[data-hash*="#inbox"] header { background-color: #4285f4 !important; }
+._in[data-hash*="#starred"] header { background-color: #FDB500 !important; }
+._in[data-hash*="#snoozed"] header { background-color: #ef6c00 !important; }
+._in[data-hash*="#search/-is%3Ainbox"] header { background-color: #0f9d58 !important; }
```
Comment (tpai) [inline in.css]: @kallepersson Fix toolbar background color, it should not change to gray when unfold the email.

**Pull request merged**